gateway-shell/src/main/java/org/apache/knox/gateway/shell/table/JDBCKnoxShellTableBuilder.java

```diff
@@ -107,15 +109,12 @@ public KnoxShellTable sql(String sql) throws IOException, SQLException {
     return this.table;
   }
 
-  public Connection createConnection() throws SQLException {
-    Connection con = null;
-    if (username != null && pass != null) {
-      con = DriverManager.getConnection(connectionUrl, username, pass);
-    }
-    else {
-      con = DriverManager.getConnection(connectionUrl);
+  private Connection createConnection() throws SQLException {
+    if (StringUtils.isNotBlank(username) && StringUtils.isNotBlank(pass)) {
```

Comment:
should a `""` password be allowed? (I have no idea if that is valid)

Reply:
it makes sense; I've updated the code